### PR TITLE
🧹 update example with cnspec provisioner

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: examples/packer-aws/ubuntu-18.04.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
       ]
     },
     {
-      "type": "mondoo",
+      "type": "cnspec",
       "on_failure": "continue",
       "mondoo_config_path": "{{user `mondoo_config_path`}}",
```
Comment (scottford-io) [inline examples/packer-aws/ubuntu-18.04.json:40]: I don't think we need this anymore

File: examples/packer-aws/ubuntu-20.04.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
       ]
     },
     {
-      "type": "mondoo",
+      "type": "cnspec",
       "on_failure": "continue",
       "mondoo_config_path": "{{user `mondoo_config_path`}}",
```
Comment (scottford-io) [inline examples/packer-aws/ubuntu-20.04.json:40]: We should remove this.

**Review by scottford-io — CHANGES_REQUESTED** [review]
@chris-rock thanks for updating these. I think we should remove the platform-specific configs from the examples now that we use `cnspec` and those inputs only work when connected to Mondoo Platform. 

Also, I don't think we need the `mondoo_config_path` input anymore.

File: examples/packer-aws/amazon-linux-2.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
       ]
     },
     {
-      "type": "mondoo",
+      "type": "cnspec",
       "on_failure": "continue",
       "mondoo_config_path": "{{user `mondoo_config_path`}}",
```
[the remark this answers is not in the document]
Comment (chris-rock) [reply examples/packer-aws/amazon-linux-2.json:40]: I think we should remove this from default examples

**Comment by chris-rock** [comment]
@scottford-io great suggestion. I removed `mondoo_config_path` from the examples since they are not required anymore.

**Review by scottford-io — APPROVED** [review]
Thanks @chris-rock !!! 👍

**Pull request merged**